integration-tests/tests/src/setup-globals.ts

```diff
@@ -55,7 +66,7 @@ describe("Test Harness", function (this: Mocha.Suite) {
    * @see [typings.d.ts](./typings.d.ts) for documentation.
    */
   function longTimeout(this: Mocha.Context | Mocha.Suite) {
-    this.timeout(environment.longTimeout || DEFAULT_LONG_TIMEOUT); // 30 seconds
+    this.timeout(environment.longTimeoutMs || DEFAULT_LONG_TIMEOUT); // 30 seconds
```

Comment:
There's a comment where the variable is defined already 👍 
```suggestion
    this.timeout(environment.longTimeoutMs || DEFAULT_LONG_TIMEOUT);
```